app/templates/components/hyper-table.hbs

```diff
@@ -52,7 +63,8 @@
           <div><i class="fa fa-caret-right"></i></div>
         </div>
         {{#each _fieldCategories as |category|}}
-          <div class="field-category {{if (eq _activeFieldCategory category.key) 'field-category--active'}}" {{action "setFieldCategory" category.key}}>
+          <div class="field-category {{if (eq _activeFieldCategory category.key) 'field-category--active'}}"
+               data-control-name="field_category_toggle_{{ category.key }}" {{action "setFieldCategory" category.key}}>
```

Comment:
can you use `concat` here too ?

Reply:
I'm not in a component evaluation, how can I ?

Reply:
Somethink like that : `name={{concat "field_category_toggle_" category.key)`

Reply:
:exploding_head: